### ISSUE TITLE
Text analytics API not working

### ISSUE THREAD

**Issue by biswapm** [Power-Maverick/PCF-Controls#124]
Azure text analytics API not giving any json response, giving 202 code, do I need to setup it SPN or pass bearer token to authorize.

**Comment by Power-Maverick** [comment]
Have you provided the correct Subscription Key & Endpoint URL as shown in this video - https://www.youtube.com/watch?v=Vksv9q2uMZw&t=120s

**Comment by biswapm** [comment]
yes I have given the correct key, it is giving me 202 accepted response but not showing any json response. Do you see any scenarios to use this PCF than text analytics connectors https://learn.microsoft.com/en-us/connectors/cognitiveservicestextanalytics/ ?

**Comment by Power-Maverick** [comment]
Closing it as stale
